gtfs_realtime_translators/translators/septa_trolley_lines.py

```diff
@@ -7,17 +7,16 @@
 
 
 class SeptaTrolleyLineTranslator:
-    def __init__(self, stop_id=None):
-        if stop_id is None:
-            raise ValueError('stop_id is required.')
-
-        self.stop_id = stop_id
 
     def __call__(self, data):
         json_data = json.loads(data)
         entities = []
         for data in json_data["data"]["predictionsData"]:
-            trip_updates = self.__make_trip_updates(data, self.stop_id)
+            stop_id = data.get("stopId", None)
```

Comment:
we do this a lot (lines 15-17)
Can we introduce some kind of helper that does some validation & error handling plus returns the value if it's defined?

```
def json_contains_required_field(field, json):
  if field is none: 
   ... raise error
  return field
```

Reply:
Can not include jason as a part of method as most of them are in constructor and being passed as argument while creating transformer. Adding check for just field check.